Sources/StreamChatUI/ChatMessageList/MessageComposer/ChatMessageComposerView.swift

```diff
@@ -173,8 +169,16 @@ open class _ChatMessageComposerView<ExtraData: ExtraDataTypes>: _View, ThemeProv
         centerLeftContainer.addArrangedSubview(attachmentButton)
         centerLeftContainer.addArrangedSubview(commandsButton)
         centerLeftContainer.addArrangedSubview(shrinkInputButton)
+
+        dismissButton.widthAnchor.pin(equalToConstant: 24).isActive = true
+        dismissButton.heightAnchor.pin(equalToConstant: 24).isActive = true
```

Comment:
nit: `dismissButton.heightAnchor.pin(equalTo: dismissButton.heightAnchor).isActive = true`

Reply:
I don't like that much because you have to type more code than just doing what I did, IMO it looks cleaner than the code above, and the benefits of that are really not worth it